guacamole/src/main/webapp/translations/cz.json

```diff
@@ -0,0 +1,691 @@
+    "FIELD_PLACEHOLDER_FILTER": "@:APP.FIELD_PLACEHOLDER_FILTER",
+    "HELP_CLIPBOARD": "Zde se zobrazí text zkopírovaný / oříznutý v Guacamole. Změny níže uvedeného textu ovlivní vzdálenou schránku.",
+    "HELP_INPUT_METHOD_NONE": "Není použita žádná metoda vstupu. Vstup z klávesnice je přijímán z připojené fyzické klávesnice.",
+    "HELP_INPUT_METHOD_OSK": "Zobrazte a přijměte vstup z vestavěné klávesnice Guacamole na obrazovce. Klávesnice na obrazovce umožňuje zadávat kombinace kláves, které jinak mohou být nemožné (například Ctrl-Alt-Del).",
+    "HELP_INPUT_METHOD_TEXT": "Povolit psaní textu a emulovat události klávesnice na základě zadaného textu. To je nezbytné pro zařízení, jako jsou mobilní telefony, které nemají fyzickou klávesnici.",
+    "HELP_MOUSE_MODE": "Určuje, jak se bude vzdálená myš chovat s ohledem na dotyky.",
+    "HELP_MOUSE_MODE_ABSOLUTE": "Tap to click. Kliknutí nastane v místě dotyku.",
+    "HELP_MOUSE_MODE_RELATIVE": "Přetažením myši posuňte ukazatel myši a klepněte na tlačítko. Kliknutí nastane v místě ukazatele.",
+    "HELP_SHARE_LINK": "Aktuální připojení je sdíleno a může k němu přistupovat kdokoli s následujícím {LINKS, plural, one{link} other{links}}:",
+    "INFO_CONNECTION_SHARED": "Toto připojení je nyní sdíleno.",
+    "INFO_NO_FILE_TRANSFERS": "Žádné přenosy souborů.",
+    "NAME_INPUT_METHOD_NONE": "Žádné",
+    "NAME_INPUT_METHOD_OSK": "Na obrazovce, klávesnice",
+    "NAME_INPUT_METHOD_TEXT": "Textový vstup",
+    "NAME_KEY_CTRL": "Ctrl",
+    "NAME_KEY_ALT": "Alt",
+    "NAME_KEY_ESC": "Esc",
+    "NAME_KEY_TAB": "Tab",
+    "NAME_MOUSE_MODE_ABSOLUTE": "Dotyková obrazovka",
+    "NAME_MOUSE_MODE_RELATIVE": "Touchpad",
+    "SECTION_HEADER_CLIPBOARD": "Schránka",
+    "SECTION_HEADER_DEVICES": "Zařízení",
+    "SECTION_HEADER_DISPLAY": "Zobrazení",
+    "SECTION_HEADER_FILE_TRANSFERS": "Přenos souborů",
+    "SECTION_HEADER_INPUT_METHOD": "Metoda vstupu",
+    "SECTION_HEADER_MOUSE_MODE": "Mód emulace myši",
+    "TEXT_ZOOM_AUTO_FIT": "Automaticky přizpůsobit prohlížeč oknu",
+    "TEXT_CLIENT_STATUS_IDLE": "Nečinný",
+    "TEXT_CLIENT_STATUS_CONNECTING": "Připojuji ke Guacamole...",
+    "TEXT_CLIENT_STATUS_DISCONNECTED": "Byl jste odpojen.",
+    "TEXT_CLIENT_STATUS_UNSTABLE": "Síťové spojení ke  Guacamole serveru se zdá nestabilní.",
+    "TEXT_CLIENT_STATUS_WAITING": "Připojen ke Guacamole. Čekání na odpověď...",
+    "TEXT_RECONNECT_COUNTDOWN": "Znovu připojuji  {REMAINING} {REMAINING, plural, one{second} other{seconds}}...",
+    "TEXT_FILE_TRANSFER_PROGRESS": "{PROGRESS} {UNIT, select, b{B} kb{KB} mb{MB} gb{GB} other{}}",
+    "URL_OSK_LAYOUT": "layouts/en-us-qwerty.json"
+  },
+  "DATA_SOURCE_DEFAULT": {
+    "NAME": "Výchozí (XML)"
+  },
+  "FORM": {
+    "FIELD_PLACEHOLDER_DATE": "YYYY-MM-DD",
```

Comment:
Based on some recent experience with this issue, this may need to be `YYYY-MM-dd` - AngularJS uses `dd` as the placeholder for a zero-padded date, and someone else was running into issues when they tried to use `DD` in this type field.

Reply:
I changed it to dd